server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -213,7 +219,13 @@ public void headersShouldBeUnfoldedAndDecoded() throws Exception {
             .preview("(Empty)")
             .textBody(Optional.of(""))
             .htmlBody(Optional.empty())
+            .isAnswered(false)
```

Comment:
We have another comment from Matthieu that Jmap should not lean on Flags anymore so it's not OK with .flags(Flags flags) for both is* and keywords.
As my understanding, MessageFactory is the case we convert from imap Message to jmap Messasge and sooner or later we will not support is*. I would like to keep as it is such as imap message's flags can convert to is* or keywords depend on what we want to support.
WDYT?